### PR TITLE
Janus example

### PR DESCRIPTION
**Pull request by davidkornel**
This PR adds the Janus WebRTC server example. It includes a README file explaining all the steps to install Janus and STUNner into Kubernetes. It also comes with two yaml manifests containing configuration for the STUNner ecosystem and deployment manifests for the Janus ecosystem.

Self built Janus images are pushed under the l7mp org on Docker Hub. [Janus Web](https://hub.docker.com/r/l7mp/janus-web) [Janus Gateway](https://hub.docker.com/r/l7mp/janus-gateway)

TODO: Figure about the deployment in Kubernetes. The missing figure is highlighted in the README. 

Tested :heavy_check_mark:

Someone else should test it before merging.

### CODE REVIEW

File: docs/examples/janus/README.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+Authentication type: static, username/password: user-1/pass-1
+Listeners:
+  - Name: stunner/udp-gateway/udp-listener
+    Protocol: TURN-UDP
+    Public address:port: 34.118.88.91:3478
+    Routes: [stunner/iperf-server]
+    Endpoints: [10.76.1.4, 10.80.4.47]
+```
+
+Note that Janus itself will not use STUNner (that would amount to a less efficient [symmetric ICE mode](../../DEPLOYMENT.md)); with the above configuration we are just telling Janus Web to instruct its clients to use STUNner to reach the Janus Gateway server.
```
Comment (levaitamas) [inline docs/examples/janus/README.md:173]: please clarify: Janus will not use STUNner _as a TURN server_

File: docs/examples/janus/README.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+- a Janus Gateway server,
+- a web server serving the landing page using [Janus Web Demos](https://github.com/meetecho/janus-gateway/tree/master/html)
+- a cluster issuer for the TLS certificates,
+- an Ingress resource to terminate the secure connections between your browser and the Kubernetes cluster.
+
+Wait until all pods become operational and jump right into testing!
+
+## Test
+
+After installing everything, execute the following command to retrieve the URL of your fresh Janus demo app:
```
Comment (levaitamas) [inline docs/examples/janus/README.md:198]: nitpicking: fresh -> freshly deployed?

File: docs/examples/janus/README.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+
+```console
+echo client-$INGRESSIP.nip.io
+```
+
+Copy the URL into your browser, and now you should be greeted with the About page. On the landing page navigate to the Video call plugin demo (`/demos/videocall.html`). Duplicate the tab and register two users in the system and make a call.  If everything is set up correctly, you should be able to connect to a room. If you repeat the procedure in a separate browser tab you can enjoy a nice video-conferencing session with yourself, with the twist that all media between the browser tabs is flowing through STUNner and the Janus Gateway server deployed in you Kubernetes cluster.
```
Comment (levaitamas) [inline docs/examples/janus/README.md:204]: Can you please add a Help section to the very end?
An example:
```
# Help

STUNner development is coordinated in Discord, feel free to [join](https://discord.gg/DyPgEsbwzc).
```

**Review by levaitamas — COMMENTED** [review]
Nicely done, thanks! I left some minor comments. Please address them and do the fig.

**Comment by davidkornel** [comment]
@levaitamas can you check one more time? Notice that I also added the dockerfiles and added a respective section also.

**Comment by levaitamas** [comment]
LGTM. Thanks again!

**Pull request merged**